src/Equinor.ProCoSys.IPO.Query/GetInvitationsQueries/GetInvitations/GetInvitationsQueryHandler.cs

```diff
@@ -31,20 +32,58 @@ public async Task<Result<InvitationsResult>> Handle(GetInvitationsQuery request,
             // count before adding sorting/paging
             var maxAvailable = await queryable.CountAsync(token);
 
-            var enumerable = AddSorting(request.Sorting, queryable.AsEnumerable());
-            enumerable = AddPaging(request.Paging, enumerable);
+            queryable = AddSorting(request.Sorting, queryable);
+            queryable = AddPaging(request.Paging, queryable);
 
-            var orderedDtos = enumerable.ToList();
+            var orderedDtos = await queryable.ToListAsync(token);
 
             if (!orderedDtos.Any())
             {
                 return new SuccessResult<InvitationsResult>(new InvitationsResult(maxAvailable, null));
             }
 
-            return new SuccessResult<InvitationsResult>(new InvitationsResult(maxAvailable, orderedDtos));
+            var invitationIds = orderedDtos.Select(i => i.Id).ToList();
+
+            var invitationsWithIncludes = GetInvitationsWithIncludesAsync(_context, invitationIds, token).Result;
```

Comment:
any reason for not having:
var invitationsWithIncludes = awaut GetInvitationsWithIncludesAsync(_context, invitationIds, token);

(both places)